features/FieldTypeForm/user_fieldtype_edit_form.feature

```diff
@@ -0,0 +1,22 @@
+Feature: User field value edit form
+    In order to edit content of ezuser fields
+    As an integrator
+    I want the ezuser field form to implement the FieldType's behaviour
+
+Background:
+    Given a Content Type with a user field definition
+
+Scenario: The attributes of a user field have a form representation
+    When I view the edit form for this field
+    Then the edit form should contain a fieldset named after the field definition
+     And it should contain the following set of labels and input fields types:
```

Comment:
They're not "fields types", they're "input fields" types. Big difference. We have a domain clash, here, between our content model and the HTML forms domain.